src/components/Project.js

```diff
@@ -24,34 +23,18 @@ function Project(props) {
             props.index % 2 === 1 ? "right" : ""
           }`}
         >
-          <a src={props.project.deployment}>
-            <img
-              className={`project-img`}
-              src={props.project.imageURL}
-              alt={props.project.title}
-            />
-          </a>
+          <img
+            className={`project-img`}
+            src={props.project.imageURL}
+            alt={props.project.title}
+          />
         </div>
         <div className="project-text">
           <h2 className="project-title">{props.project.title}</h2>
-          <p className="project-description">{props.project.description}</p>
-          <div className="project-actions">
-            <a
-              className="deployment-btn"
-              href={props.project.deployment}
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Check it Out
-            </a>
-            <a
-              className="repo"
-              href={props.project.repo}
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              <img src={gitHub} alt="Repository" />
-            </a>
+          <div className="project-description">
+            {props.project.preview.split("\n").map((p, key) => {
+              return <p key={key}>{p}</p>;
+            })}
```

Comment:
Added so long string description from back-end can be parsed and split into different p tags for paragraph spacing.